tests/test_cosmology.py

```diff
@@ -24,27 +24,29 @@ def get_cosmologies() -> List[Tuple[Cosmology, Cosmology_AP]]:
     return cosmologies
 
 
-def test_comoving_dist():
+def test_comoving_dist(device):
     rtol = 1e-3
     atol = 0
 
-    zs = torch.linspace(0.05, 3, 10)
+    zs = torch.linspace(0.05, 3, 10, device=device)
     for cosmology, cosmology_ap in get_cosmologies():
-        vals = cosmology.comoving_distance(zs).numpy()
-        vals_ref = cosmology_ap.comoving_distance(zs).value / 1e2  # type: ignore
-        assert np.allclose(vals, vals_ref, rtol, atol)
+        cosmology.to(device=device)
 
+        vals = cosmology.comoving_distance(zs)
+        vals_ref = cosmology_ap.comoving_distance(zs.to("cpu").numpy()).value / 1e2  # type: ignore
```

Comment:
instead of `.to("cpu")` you can just call `.cpu()` which will move the object to the cpu